pnpm-lock.yaml

```diff
@@ -1,43 +1,54 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
nit: Check why the version is downgraded. Is it increase compatibility or we need to use later version locally?